app/src/main/java/com/pokemonshowdown/app/UpdateCheckTask.java

```diff
@@ -87,7 +87,10 @@ private boolean isUpdateAvailable(String currentVersion, String serverVersion) {
 
             if (server > current) {
                 return true;
+            } else if (current > server) {
+                return false;
```

Comment:
This is not a good way of doing it. We shouldn't rely on version name, but rather version code : http://developer.android.com/tools/publishing/versioning.html

This version split thing will get confusing.